Makefile

```diff
@@ -20,29 +20,25 @@ $(WASM_PACK):
 ###############################################################################
 # BUILD
 
-KCL_WASM_LIB_FILES := $(wildcard rust/**/*.rs)
-TS_SRC := $(wildcard src/**/*.tsx) $(wildcard src/**/*.ts)
-XSTATE_TYPEGENS := $(wildcard src/machines/*.typegen.ts)
+RUST_SOURCES := $(wildcard rust/**/*.rs) rust/Cargo.*
```

Comment:
This is also missing: `rust/.cargo/config.toml`, `rust/rust-toolchain.toml`, and `rust/**/Cargo.*`. Like Lee said, I would probably just go off of the entire directory.